libraries/lvgl.lib

```diff
@@ -1 +1 @@
-https://github.com/lvgl/lvgl/#7506b61527195fdc96803fe94f5ba141b6e5a9e8
\ No newline at end of file
+https://github.com/lvgl/lvgl/#a65ca267bf9e4a55d6a303aee17cfbe22d464f2b
```

Comment:
Not sure why this is changed. Previous one was correct. Please use:
https://github.com/lvgl/lvgl/#7506b61527195fdc96803fe94f5ba141b6e5a9e8